veps/vep-5.md

```diff
@@ -0,0 +1,27 @@
+# VEP-5 Vitess Bugfixes
+
+```
+VEP: 5
+Title: Vitess Bugfixes
+Author: Andres Taylor
+Reviewers: 
+Status: Proposed
+Created: 2021-05-19
+```
+
+## Abstract
+
+[VEP-1](https://github.com/vitessio/enhancements/blob/master/veps/vep-1.md) established a support lifecycle for Vitess. [VEP-2](https://github.com/vitessio/enhancements/blob/master/veps/vep-2.md) later amended parts of the
+This VEP suggests a change to how bugfixes are handled, how often releases are made, and how long they are supported for and thus replaces the Support lifecycle section in VEP-1. 
+
```

Comment:
I think we need to put in an edited version of this section from [VEP-2](https://github.com/vitessio/enhancements/blob/master/veps/vep-2.md):
```
## Versioning and Release Cadence
1. The COMPATIBILITY number will be incremented every 13 weeks. For example, the next major release will have a COMPATIBILITY number of 7.0.
2. These major releases will occur in the last week of January, April, July and October.
3. Each COMPATIBILITY number will be supported by the Open Source Vitess community for 9 months after the initial release. Support Lifecycle is described in VEP-1.
4. Patch releases will be created as needed. Patch releases will follow SemVer. The numbering scheme will be COMPATIBILITY.PATCH_NUMBER. For example, the first patch release on 7.0, if there is one, will be 7.0.1. PATCH_NUMBER will be monotonically increasing.
5. Events that trigger patch releases are major regressions in functionality or performance, and CVEs.
```